style.css

```diff
@@ -256,3 +259,80 @@
   align-items: center;
   margin-left: -2em;
 }
+
+/* form-section */
+
+.form-section {
+  display: flex;
+  flex-direction: column;
+  height: fit-content;
+  padding-top: 120px;
+}
+
+.contact-title {
+  display: flex;
+  text-align: center;
+  align-items: center;
+  padding: 5px 10px 5px 10px;
+  font-size: 2em;
+  color: #172b4d;
+}
+
+.form-container {
+  display: flex;
+  flex-direction: column;
+  padding: 30px 20px 120px 20px;
+  gap: 20px;
+}
+
+.input-form {
+  outline: none;
+  border: none;
+  border-bottom: 1px solid #dfe1e6;
+  padding: 0 0 10px 20px;
+  font-size: 15px;
+}
+
+.form-text-area {
+  background: #fbf8f7;
+  border: none;
+  padding: 20px;
+  font-size: 15px;
+}
+
+.form-button {
+  background-color: #ff6b00;
+  width: 100px;
+  height: 40px;
+  color: white;
+  border: none;
+  align-self: center;
+}
+
+.form-button:hover {
+  cursor: pointer;
+  border: 2px solid grey;
+}
```

Comment:
- [ ] You did a great job working on the button effects 🚀 However, please try to stick as much as possible to the FIGMA design:

![image](https://user-images.githubusercontent.com/77069011/205938278-46e101af-c9aa-4e0c-9cad-fb4e4c5a7eba.png)

Please, consider updating the button interactions to match the ones shown in the template. One of the main goals of this project is that you demonstrate your capability to build entire websites following as closely as possible a given design. Keep it up! 🤓 🛠️